bridges/ComicsKingdomBridge.php

```diff
@@ -41,6 +41,8 @@ public function collectData(){
 			$item['content'] = '<img src="' . $imagelink . '" />';
 
 			$this->items[] = $item;
+			$link = $page->find('div.comic-viewer-inline a', 0)->href;
```

Comment:
Notice: Trying to get property 'href' of non-object in /var/www/rss-bridge/pr2444/bridges/ComicsKingdomBridge.php on line 42
https://feed.eugenemolotov.ru/pr2444/?action=display&bridge=ComicsKingdom&comicname=mutts&format=Html

Reply:
My apologies, I had display_errors off and failed to read the error log.  As one might expect, this notice was occurring because the site is currently not working properly when going back more than 3 days of comics, as doing so results in a blank page.  I was hoping it might fix itself by now as more comics arrived, but that hasn't happened.

I have therefore changed the loop in 9fda15e to be only 3 days, but still left in the error handling, as it will still fatally fail if the site changes again, but only write a notice in the unlikely event it can't get all 3 days worth of comics.

Reply:
Nevermind. Instead of setting up display_errors, you can just create empty DEBUG file in rss-bridge root directory.